### DISCUSSION TITLE
Does giu have a file selector? It is better to call the system native

### DISCUSSION THREAD

**Discussion by Seikaijyu** [AllenDang/giu#536] [Q&A] [answered]
Does giu have a file selector? It is better to call the system native

**Comment by gucio321** [comment, marked as answer]
No, giu doesn't have one, but I was using [github.com/sqweek/dialog](https://github.com/sqweek/dialog) some time ago and it works with giu well.

**Reply by Seikaijyu** [reply]
It looks good. Thank you very much for your answer
